.nvim/lua/nvim-project-alicorn.lua

```diff
@@ -13,8 +13,7 @@ M.vscode_local_settings = {}
 ---@generic T
```

Comment:
Included this as a separate commit at the start because stylua hadn't been applied and it makes CI fail